mesh/occupations/views.py

```diff
@@ -1,3 +1,173 @@
-from django.shortcuts import render
+# Django Imports
+from django.http import JsonResponse, HttpResponse
+from django.views import View
 
-# Create your views here.
+# Model Imports
+from .models import Occupation, OccupationBridge
+from mesh.accounts.models import Account
+from mesh.profiles.models import Profile
+
+# Utilities Imports
+from ..utils.validate_data import validate_json_and_required_fields
+
+# Library Imports
+import json
+
+
+class OccupationsView(View):
+    """
+        Handles HTTP requests related to the collection of Occupation,
+        supporting GET to retrieve the list of all Occupations and
+        POST to create a new Occupation.
+    """
+    
+    def get(self, request, *args, **kwargs):
```

Comment:
Is it possible to do get_all

Reply:
These function names are based explicitly off of HTTP request methods, and since GET_ALL is not an HTTP request method, unfortunately this can't be named get_all.